app/templates/brief.html

```diff
@@ -1,188 +1,435 @@
-{% extends "_base_page.html" %}
-
-{% block page_title %}{{ brief.title }} - Digital Marketplace{% endblock %}
-
-{% block body_classes %}brief-builder brief-content{% endblock %}
-
-
-{% block breadcrumb %}
-{%
-with
-items = [
-{
-"link": url_for('.index'),
-"label": "Home"
-},
-{
-"link": url_for('.list_opportunities', framework_slug=brief.frameworkSlug),
-"label": "Opportunities"
-},
-{
-"label": "Opportunity details"
-}
-]
-%}
-{% include "toolkit/breadcrumb.html" %}
-{% endwith %}
-{% endblock %}
-
-{% block main_content %}
-
-{% if brief.status == 'closed' %}
-
+{% extends "_base_page.html" %} {% block page_title %}{{ brief.title }} - Digital Marketplace{% endblock %} {% block body_classes
```

Comment:
some repetition in this file - same templates repeated

gotta think how this can be more maintainable cause its turning into a monster